airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -280,6 +282,13 @@ def create_empty_table(self,
                 "useLegacySql": False
             }
 
+        :param encryption_configuration: [Optional] Custom encryption configuration (e.g., Cloud KMS keys).
+            **Example**: ::
+
+                encryption_configuration = {
+                    "projects/testp/locations/us/keyRings/test-kr/cryptoKeys/test-key"
```

Comment:
Is this the correct value for sure? I look at the documentation and it seems to me that it should be a dictionary with the key "kmsKeyName" 
```
      "encryptionConfiguration": { # Custom encryption configuration (e.g., Cloud KMS keys).
        "kmsKeyName": "A String", # [Optional] Describes the Cloud KMS encryption key that will be used to protect destination BigQuery table. The BigQuery Service Account associated with your project requires access to this encryption key.
      },
```
Reference: 
https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#Table
https://cloud.google.com/bigquery/docs/reference/rest/v2/EncryptionConfiguration
https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/python/latest/bigquery_v2.tables.html#patch

Reply:
@mik-laj  You are right. I have that key in the unit tests and my own end-to-end tests. But somehow I had put the wrong examples. Thanks for reviewing!

Reply:
@mik-laj All fixed. PTAL